validator/src/main/java/com/amazon/aoc/validators/LoadBalancingValidator.java

```diff
@@ -97,7 +102,7 @@ private String getSampleAppResponse() throws Exception {
     return sampleAppResponse.getTraceId();
   }
 
-  private boolean checkSpanCount(List<String> spanSet) throws Exception {
+  private boolean checkSpanCount(List<String> spanSet) {
```

Comment:
Now that I changed the collection of spans from a set to a list, do you mind changing this variable name to something else like `spanList`?

Reply:
done